src/Controller/Api/CategoryController.php

```diff
@@ -51,9 +55,42 @@ public function __construct(CategoryManagerInterface $categoryManager, FormFacto
     /**
      * Retrieves the list of categories (paginated) based on criteria.
      *
-     * @ApiDoc(
-     *  resource=true,
-     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
+     * @Operation(
+     *     tags={"/api/classification/categories"},
+     *     summary="Retrieves the list of categories (paginated) based on criteria.",
+     *     @SWG\Parameter(
+     *         name="page",
+     *         in="query",
+     *         description="Page for category list pagination",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="count",
+     *         in="query",
+     *         description="Number of categories by page",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="enabled",
+     *         in="query",
+     *         description="Enabled/Disabled categories filter",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Parameter(
+     *         name="context",
+     *         in="query",
+     *         description="Context of categories",
+     *         required=false,
+     *         type="string"
+     *     ),
+     *     @SWG\Response(
+     *         response="200",
+     *         description="Returned when successful",
+     *         @SWG\Schema(ref=@Model(type="Sonata\DatagridBundle\Pager\PagerInterface"))
```

Comment:
IMO, we could use the `::class` pseudo-constant where possible.
```suggestion
     *         @SWG\Schema(ref=@Model(type=Sonata\DatagridBundle\Pager\PagerInterface::class))
```

Reply:
This I will do in next PR (also models should be fix/improve in swagger).